src/debug.nit

```diff
@@ -3,6 +3,10 @@ import api
 redef class AppConfig
 	# The default player, if any, that will be logged on a new session
 	var debug_player: String is lazy do return value_or_default("app.debug.player", "")
+
+	init do
+		engine_map["nitc"] = new NitcEngine
```

Comment:
I'm thinking about something else:

Why engine: `nitc`?
* I would prefer `nit`, as it will be more intuitive that `nitc` in a future config file or dropdown menu
* I don't think we will use another engine like niti or nitcg?